src/version2/task.ts

```diff
@@ -78,7 +78,7 @@ export class Task {
       url: `/tasks/${parameters.id}`,
       method: 'GET',
       params: {
-        'body-format': parameters['body-format'],
+        'body-format': parameters.bodyFormat,
```

Comment:
I getting an error when try to build the lib. Could you please check how 
[Augustus Thomas](https://github.com/augustus-thomas) did [similar changes](https://github.com/MrRefactoring/confluence.js/pull/85) and fix it?

![The error](https://github.com/MrRefactoring/confluence.js/assets/10329528/3cae8f4d-5636-4cac-9ed8-09a75444abcc)